### PR TITLE
User guide first version

### PR DESCRIPTION
**Pull request by deralexxx**
This is a first version of the user guide. It covers:

- most of the tsctl commands
- Concepts (to be improved)
- Searching basics
- the demo page

It will be for sure improved in the future but I think it is worth for a first merge. Happy to get feedback

### CODE REVIEW

File: docs/Users-Guide.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+```
+tsctl runserver -c /etc/timesketch.conf
+```
+
+
+### Start timesketch
+
+Will start the timesketch server
+
+Comand:
```
Comment (berggren) [inline docs/Users-Guide.md]: Command

Comment (berggren) [reply docs/Users-Guide.md]: Same for the rest of the file. s/Comand/Command

Comment (deralexxx) [reply docs/Users-Guide.md]: ack, will change those

**Comment by deralexxx** [comment]
Ready to merge @berggren

**Comment by berggren** [comment]
Merged, thanks for contributing!

**Pull request merged**